### PR TITLE
Deleted supported versions table

### PR DESCRIPTION
**Pull request by dmuntima**
### Type of change

- Documentation

### Description

Deleted the Supported Versions table and rewrote the text accordingly.

### CODE REVIEW

File: documentation/book/getting-started.adoc [hunk 1]
```diff
@@ -14,24 +14,9 @@ This guide expects that an {ProductPlatformName} cluster is available and the
...
-
-|{OpenShiftLongName}
-|{OpenShiftVersion}
-
-|Apache Kafka
-|2.0.0
-
-|====
+ifdef::Downloading[]
+{ProductName} is based on Strimzi 0.8.0. This chapter describes the procedures to deploy AMQ Streams on {OpenShiftName} {OpenShiftVersion}.
```
Comment (scholzj) [inline documentation/book/getting-started.adoc]:
```suggestion
{ProductName} is based on Strimzi 0.8.0. This chapter describes the procedures to deploy {ProductName} on {OpenShiftName} {OpenShiftVersion}. 
```

File: documentation/book/getting-started.adoc [hunk 2]
```diff
@@ -14,24 +14,9 @@ This guide expects that an {ProductPlatformName} cluster is available and the
...
-endif::Kubernetes[]
-
-|{OpenShiftLongName}
-|{OpenShiftVersion}
-
-|Apache Kafka
-|2.0.0
-
-|====
+ifdef::Downloading[]
```
Comment (scholzj) [inline documentation/book/getting-started.adoc]: Should this be really `ifdef::Downloading[]`? I think that will render it only upstream. I think this should be `ifndef::Downloading[]` to show it only downstream, or?

File: documentation/book/getting-started.adoc (hunk 1 above)
Comment (scholzj) [inline documentation/book/getting-started.adoc]: I wonder if the version should be hardcoded like this. That will make it easy to forget to update it.

File: documentation/book/getting-started.adoc (hunk 2 above)
Comment (laidan6000) [reply documentation/book/getting-started.adoc]: That's my understanding too. Those variables are confusingly named!

File: documentation/book/getting-started.adoc (hunk 1 above)
Comment (laidan6000) [reply documentation/book/getting-started.adoc]: I think this should say: "{ProductName} is based on Apache Kafka". This should be upstream only.

File: documentation/book/getting-started.adoc (hunk 2 above)
Comment (laidan6000) [reply documentation/book/getting-started.adoc]: So, this change should be **downstream** only.

**Comment by dmuntima** [comment]
@scholzj @laidan6000 I am going to add the "StrimziVersion" attribute only in the downstream attributes file as it is not applicable here.

**Comment by dmuntima** [comment]
Due to time constraints, merging the PR now.

**Pull request merged**